packages/styles/panel.css

```diff
@@ -33,18 +100,11 @@ fieldset.Panel {
   top: var(--space-large);
 }
 
-.Panel--collapsed {
-  --panel-padding: 12px;
-}
-
-.Panel--collapsed .Panel__Heading {
-  margin: 0 0 12px 0;
-  font-size: var(--text-size-normal);
-}
```

Comment:
This can't be removed as it would make it a breaking change. Even though we do plan on removing these values, we need to go through a deprecation period first.